README.md

```diff
@@ -201,6 +201,7 @@ The optional `options` object may contain:
 - `createIfMissing` (boolean, default: `true`): If `true`, create an empty database if one doesn't already exist. If `false` and the database doesn't exist, opening will fail.
 - `errorIfExists` (boolean, default: `false`): If `true` and the database already exists, opening will fail.
 - `passive` (boolean, default: `false`): Wait for, but do not initiate, opening of the database.
+- `allowMultiThreading` (boolean, default: `false`): Allow multiple threads to access the database. This is only relevant when using [`node.js worker_threads`](https://nodejs.org/api/worker_threads.html)
```

Comment:
```suggestion
- `allowMultiThreading` (boolean, default: `false`): Allow multiple threads to access the database. This is only relevant when using [worker threads](https://nodejs.org/api/worker_threads.html)
```

No need to qualify it, because we officially only support Node.js atm, and not e.g. bun or Deno.

Reply:
Updated